File: src/main/java/com/company/payroll/model/JwtTokenProperties.java
package com.company.payroll.model;

import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Component
@ConfigurationProperties("jwt.config")
public class JwtTokenProperties {
	private String key;
	private long ttl;
}

File: kotlin-payroll/src/main/kotlin/com/company/payroll/model/JwtTokenProperties.kt
package com.company.payroll.model

import org.springframework.beans.factory.annotation.Value
import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@Component
@ConfigurationProperties("jwt.config")
data class JwtTokenProperties(
    @Value("jwt.config.key")
    var key: String,
    @Value("jwt.config.ttl")
    var ttl: String
)
